server/form-pages/apply/reasons-for-placement/basic-information/reasonForShortNotice.ts

```diff
@@ -35,7 +35,6 @@ export default class ReasonForShortNotice implements TasklistPage {
       this.title = 'Emergency application'
       this.question = 'What was the reason for submitting this application 7 days or less before the AP is needed?'
     } else {
-      this.title = 'Short notice application'
```

Comment:
Given the template is calling `this.title` in a `h1`, will this change lead to an empty `h1` for short notice applications?

You should be able to get round this by making a couple of changes in the template:
1. wrap the `h1` in a conditional block to check if there's a page title
2. add `isPageHeading: !this.title` (not 100% sure if that's Nunjucks-compliant logic) to the `legend`